tool/dartpad_picker/web/dartpad_picker_main.dart

```diff
@@ -33,18 +33,18 @@ void main() {
   DartPadPicker(dartPadHost, select, snippets, frameId: 'try-dart-pad');
 }
 
-Snippet _createSnippet(HtmlElement root, String name) => Snippet(
-      name,
-      _src(
-        root,
-        '#try-dart-${name.toLowerCase().replaceAll(' ', '-')}',
-      ),
-    );
-
-String _src(HtmlElement root, String id,
-        [String fallback =
-            "/* Can't load example sources. Please refresh the page. */"]) =>
-    root.querySelector(id)?.innerText ?? fallback;
+// Snippet _createSnippet(HtmlElement root, String name) => Snippet(
+//       name,
+//       _src(
+//         root,
+//         '#try-dart-${name.toLowerCase().replaceAll(' ', '-')}',
+//       ),
+//     );
+//
+// String _src(HtmlElement root, String id,
+//         [String fallback =
+//             "/* Can't load example sources. Please refresh the page. */"]) =>
+//     root.querySelector(id)?.innerText ?? fallback;
```

Comment:
remove?

Reply:
Good point, I've removed the commented out code. Thanks!